### ISSUE TITLE
do not exit when vnc server is in process. Need to kill it: vncserver -kill :1 everytime

### ISSUE THREAD

**Issue by IvanRave** [corbinlc/gnuroot#38]
Tested on Android4.4
wheezyX package

**Comment by cedric-vincent** [comment]
Hello,

This is an expected behavior: PRoot stays in foreground (ie. does not exit) until all processes have exited, even if remaining processes are daemons (ie. run in background).  This issue is related to this one: https://github.com/cedric-vincent/PRoot/issues/11

@IvanRave and @corbinlc, what kind of behavior is more desirable:
- a button (or something similar) in GNURoot to kill all processes at once; or
- move GNURoot/PRoot in background when all remaining processes are daemons; or
- all of the above :)

Regards,
Cédric.

**Comment by IvanRave** [comment]
yes, current behavior - it is good behavior (not needed any special buttons)
I just don't want to run VNC by default (and kill it everytime)

**Comment by corbinlc** [comment]
I understand the feature request.  I have big changes coming (not in the next release, it will be more incremental, but soon).
